endpoints/openrtb2/sample-requests/invalid-whole/format-empty-object.json

```diff
@@ -1,6 +1,6 @@
 {
-  "message": "Invalid request: Request imp[0].banner.format[0] should define *either* {w, h} (for static size requirements) *or* {wmin, wratio, hratio} (for flexible sizes) to be non-zero.\n",
-  "requestPayload": {
+  "description": "Banner does not define witdth nor height",
```

Comment:
Nitpick: witdth is mispelled. Consider changing this message slightly to distinguish it from `format-empty-array.json`.

Reply:
Both corrected